drakrun/drakrun/util.py

```diff
@@ -133,6 +133,37 @@ def safe_delete(file_path) -> bool:
         return False
 
 
+def try_subprocess(list_args: list, msg: str, reraise=True, debug=True, **kwargs) -> subprocess.CompletedProcess:
+
+    # setup default parameters to be passed, can be overrided by supplying kwargs
+    if kwargs.get('stdout') is None:
+        kwargs['stdout'] = subprocess.PIPE
+    elif kwargs.get('stdout') == subprocess.STDOUT:
+        del kwargs['stdout']
+
+    if kwargs.get('stderr') is None:
+        kwargs['stderr'] = subprocess.PIPE
+    if kwargs.get('check') is None:
+        kwargs['check'] = True
+
+    try:
+        sub = subprocess.run(list_args, **kwargs)
+    except (FileNotFoundError, TypeError) as e:
+        if debug:
+            logging.debug("arguments to subprocess")
+            logging.debug(list_args)
+            logging.debug(msg)
```

Comment:
I'm not sure if I understand the idea behind debug flag.
Debug log level shold be enough to handle this. :thinking:

Reply:
Ah, makes sense, I will remove it